images/ckan/2.9/setup/app/start_ckan.sh

```diff
@@ -12,6 +12,14 @@ then
     done
 fi
 
+if grep -E "beaker.session.secret ?= ?$" ckan.ini
```

Comment:
You need to unset the default value generated when building the image (like [here](https://github.com/okfn/docker-ckan/compare/fresh-beaker-key#diff-e83d23b6fa59cf9e95a9e6087ff96669ff6b90467e66de4bc3df2d19d0490f49R78)), otherwise this condition will never be met

Reply:
Also it would be good to encourage users to set their own env vars for these secrets so they can persist across deployments, by adding some placeholders to the [env file](https://github.com/keitaroinc/docker-ckan/blob/master/compose/2.9/.ckan-env). Here's the [example](https://github.com/okfn/docker-ckan/compare/fresh-beaker-key#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR9) from the okfn images